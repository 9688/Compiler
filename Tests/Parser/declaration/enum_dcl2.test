enum T;
enum {A = 4+5/2};