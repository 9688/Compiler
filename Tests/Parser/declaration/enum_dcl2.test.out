<--------TABLES-------->
<---TABLE--->
VARS::

TYPES::
char char

double double

int int

void void

T ENUM T {
}

